packages/smooth_app/lib/pages/product/product_question_answers_options.dart

```diff
@@ -0,0 +1,104 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:openfoodfacts/model/Insight.dart';
+import 'package:openfoodfacts/model/RobotoffQuestion.dart';
+import 'package:smooth_app/generic_lib/design_constants.dart';
+
+Color _yesBackground = Colors.green;
+Color _noBackground = Colors.red;
+const Color _maybeBackground = Colors.white;
+const Color _yesNoTextColor = Colors.white;
+Color _maybeTextColor = Colors.grey.shade700;
```

Comment:
Can you move them inside the` ProductQuestionAnswersOptions` also all of them could be const, right?